esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -114,3 +148,71 @@ var RefreshButton = React.createClass({
     </button>;
   },
 });
+
+
+// Modified from react-json-table example code.
+// Required props: rows, saveState (with sort and greyed attributes), settings (with header attribute)
```

Comment:
Putting this in a PropTypes will make React automatically check it for you, and also make it more readable.